snmp/datadog_checks/snmp/snmp.py

```diff
@@ -26,24 +20,34 @@
 from .commands import snmp_bulk, snmp_get, snmp_getnext
 from .compat import read_persistent_cache, total_time_to_temporal_percent, write_persistent_cache
 from .config import InstanceConfig, ParsedMetric, ParsedMetricTag, ParsedTableMetric
-from .utils import OIDPrinter, get_profile_definition, oid_pattern_specificity, recursively_expand_base_profiles
-
-# Additional types that are not part of the SNMP protocol. cf RFC 2856
-CounterBasedGauge64, ZeroBasedCounter64 = builder.MibBuilder().importSymbols(
-    'HCNUM-TC', 'CounterBasedGauge64', 'ZeroBasedCounter64'
+from .exceptions import PySnmpError
+from .types import (
```

Comment:
The naming of this module is a bit confusing:  In the base check change (#5965) the `types.py` module contains classes used for type annotations, but in this case they are for explicit use.

Maybe rename from `types.py` to `snmp_proxy.py` or something similar?

Reply:
@mgarabed I hear you. How about `models.py`? That way I can also bring the [`OID` helper class](https://github.com/DataDog/integrations-core/pull/5531/files#diff-65fa2be8e374f641f9840b88837ac74dR15) into here later on.

Reply:
Sure, that would work nicely.

Reply:
:+1: Changed in 10f6a5c.